### PR TITLE
Use OBS Attribute for ignored issues

### LINKED ISSUE #1403: totest-manager: put issue file to dashboard
the issue file is currently local which makes it hard to run ttm locally. Should be stored in the dashboard too.

### PR DESCRIPTION
**Pull request by coolo**
Fixes #1403

### CODE REVIEW

**Comment by coolo** [comment]
This also includes #1541

File: totest-manager.py [hunk 1]
```diff
@@ -257,17 +274,28 @@ def overall_result(self, snapshot):
...
-                if not ignored:
+                if ignored:
+                    if labeled:
+                        text = 'Ignored issue' if len(refs) > 0 else 'Ignored failure'
+                        # remove flag - unfortunately can't delete comment unless admin
+                        data = {'text': text}
+                        self.openqa.openqa_request(
+                            'PUT', 'jobs/%s/comments/%d' % (job['id'], labeled), data=data)
+
+                    logger.info("job %s failed, but was ignored", jobname)
```
Comment (okurz) [inline totest-manager.py]: Could you extend this to "job %s failed, but was ignored due to '@ttm ignore' comment in job" to be explicit?

File: totest-manager.py [hunk 2]
```diff
@@ -62,15 +63,31 @@ def __init__(self, project, dryrun=False, api_url=None, openqa_server='https://o
...
-        if os.path.isfile(self.issuefile):
-            with open(self.issuefile, 'r') as f:
-                for line in f.readlines():
-                    self.issues_to_ignore.append(line.strip())
+        self.load_issues_to_ignore()
         self.project_base = project.split(':')[0]
         self.update_pinned_descr = False
 
+    def load_issues_to_ignore(self):
+        url = self.api.makeurl(['source', self.project, '_attribute', 'OSRT:IgnoredIssues'])
```
Comment (okurz) [inline totest-manager.py]: Does this also exist in IBS? We are about to use ttm for SLE as well so I hope this does not break it

Comment (coolo) [reply totest-manager.py]: it could exist - it doesn't yet

File: totest-manager.py (hunk 1 above)
Comment (coolo) [reply totest-manager.py]: we don't know at this point if the comment contained any action - or if this was a carry over who lead to the ignore. So no.

**Comment by coolo** [comment]
Actually on the way to the office one thing crossed my mind: we won't see any comment from ttm in those jobs without bug reference. Because ttm only comments 'label:unknown_failure' once there is a issue reference from carry over or reviewer. And those labels are then updated once the issue is ignored. Adding labels for ignored *jobs* is actually a different thing then.

**Pull request merged**